### PR TITLE
Version 0.7.0

### PR DESCRIPTION
**Pull request by musicinmybrain**
As requested: https://github.com/encode/databases/pull/513#issuecomment-1356445589

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,9 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.6.3 (Dec 17th, 2022)
```
Comment (Kludex) [inline CHANGELOG.md:7]:
```suggestion
## 0.6.3 (Dec 18th, 2022)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,9 +4,16 @@ All notable changes to this project will be documented in this file.
...
+
+### Fixed
+
+* Fixed breaking changes in SQLAlchemy cursor; supports `>=1.4.42,<1.5` (#513)
+* Wrapped types in typing.Optional where applicable (#510)
```
Comment (Kludex) [inline CHANGELOG.md:12]:
```suggestion
* Wrapped types in `typing.Optional` where applicable (#510).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,9 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.6.3 (Dec 17th, 2022)
+
+### Fixed
+
+* Fixed breaking changes in SQLAlchemy cursor; supports `>=1.4.42,<1.5` (#513)
```
Comment (Kludex) [inline CHANGELOG.md:11]:
```suggestion
* Fixed breaking changes in SQLAlchemy cursor; supports `>=1.4.42,<1.5` (#513).
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,9 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.6.3 (Dec 18th, 2022)
```
Comment (Kludex) [inline CHANGELOG.md:7]:
```suggestion
## 0.7.0 (Dec 18th, 2022)
```

File: databases/__init__.py [hunk 5]
```diff
@@ -1,4 +1,4 @@
 from databases.core import Database, DatabaseURL
 
-__version__ = "0.6.2"
+__version__ = "0.6.3"
```
Comment (Kludex) [inline databases/__init__.py:3]:
```suggestion
__version__ = "0.7.0"
```

**Review by Kludex — APPROVED** [review]

**Comment by Kludex** [comment]
Comparison: https://github.com/encode/databases/compare/0.6.2...master

**Comment by musicinmybrain** [comment]
Awesome. Thanks!

**Pull request merged**